bio-formats/index.html

```diff
@@ -58,7 +58,7 @@ <h2>Our Community</h2>
         </div>
         <hr>
         <div class="row column text-center">
-            <p>Bio-Formats is a community project and we welcome <a href="{{ site.baseurl }}/contributors/">your input</a>. You can find guidance on <a href="https://docs.openmicroscopy.org/bio-formats/{{ site.bf.version }}/about/bug-reporting.html" target="_blank">reporting a bug</a>, upload files to our <a href="http://qa.openmicroscopy.org.uk/qa/upload/" target="_blank">QA system</a> for testing, and contact us via our mailing lists or forums. Further information about how the OME team works and how you can contribute to our projects is in the <a href="https://docs.openmicroscopy.org/contributing/" target="_blank">Contributing Developer Documentation</a>.</p>
+            <p>Bio-Formats is a community project and we welcome <a href="{{ site.baseurl }}/contributors/">your input</a>. You can find guidance on <a href="https://docs.openmicroscopy.org/bio-formats/{{ site.bf.version }}/about/bug-reporting.html" target="_blank">reporting a bug</a>, upload files to our <a href="https://www.openmicroscopy.org/qa2//qa/upload/" target="_blank">QA system</a> for testing, and contact us via our mailing lists or forums. Further information about how the OME team works and how you can contribute to our projects is in the <a href="https://docs.openmicroscopy.org/contributing/" target="_blank">Contributing Developer Documentation</a>.</p>
```

Comment:
in the replaced url there is a double-slash https://www.openmicroscopy.org/qa2//qa/upload/

Also, is it what we are supporting ? The service seems to be non-functional. Maybe this is a point to consult with @dgault ?